### ISSUE TITLE
Accessibility/Usability: Overlay Width on Mobile

### ISSUE THREAD

**Issue by neoinsight** [wet-boew/GCWeb#1157]
@shawnthompson @touficS @pcwsmith @dk-tbs

**WIDTH OF OVERLAY**

Based on testing evidence, @matyeo recommended a wider default width for the right and left panels, as in mobile view, they are ridiculously skinny and the text breaks into just a few words.  The 60% width looks good on mobile, but it messes up the screenreader participants, because since they can't see the panel, they tap outside it right after opening, and the dang thing closes again! 
See both these painful videos: 
P1 VoiceOver iPhone Right Panel Problem: https://www.youtube.com/watch?v=a5Wkn3EEOSY
P2 VoiceOver iPhone Right Panel Problem: https://www.youtube.com/watch?v=UIADLRMJZVw

I tried it on a computer screenreader and Safari VoiceOver read out the entire panel as soon as I opened it, so while they can accidentally close it, computer screenreader users wouldn't likely have the 'close before I heard it' problem. 

DISCUSSION: If on mobile, the panel opened at a wider width, like 90%, these iPhone VoiceOver users would be less likely to be able to tap outside the panel and cause it to close without their knowledge. This wouldn't be bad for sighted Mobile users either - they would simply be more likely to use the Close button (or as Mike said, try to swipe the panel off). I would need some help with coding to make the prototype in the form on this page work that way (60% width on desktop, 90% on mobile), but it should be worth a try. 

Note that while not many GC folks are using overlays yet, we are struggling with the usability and accessibility of the Details/Summary, and hoping overlays will solve some of those issues. But they have to work properly first...

Thoughts? Thanks

**Comment by kodecount** [comment]
It is very common for overlays on mobile (especially with left/right overlays) to take 90% sometimes 100% of the screen.

**Comment by neoinsight** [comment]
@kodecount Agree, but 60% worked best for help on a form because they could still see some context.
